src/kirby/components/list/list.component.ts

```diff
@@ -66,4 +101,65 @@ export class ListComponent implements OnInit {
   onLoadOnDemand(event?: LoadOnDemandEventData) {
     this.listHelper.onLoadOnDemand(this, event);
   }
+
+  getClasses(current: any) {
```

Comment:
:fire: Move this entire section (`getClasses`, `determineClasses`, `getClassesForItem`) to helper/utility class and only apply for `.tns` (web has this build into css pseudo selectors)

Reply:
Maybe it's just me, but I can't see how any CSS-(pseudo)selector can be applied to determine "top-most" row of a section, when the "rows" and "sections headers" are all rendered in a flat li-structure?